test/integration/verify/controls/aws_sqs_queues.rb

```diff
@@ -19,4 +19,20 @@
     it           { should exist }
     its ('arns') { should include arn}
   end
+
+  aws_regions.region_names.each do |region|
+    aws_sqs_queues(aws_region: region).queue_urls.each do |queue_url|
+      describe.one do
+        describe aws_sqs_queue(aws_region: region, queue_url: queue_url) do
+          it { should exist }
+          its('kms_master_key_id') { should_not be_nil }
+        end
+
+        describe aws_sqs_queue(aws_region: region, queue_url: queue_url) do
+          it { should exist }
+          its('kms_master_key_id') { should be_nil }
+        end
```

Comment:
is this second describe needed?

Reply:
This PR is just extending the original SQS Queue resource to add the kms_master_key_id, there already exists a terraform setup for SQS queue without this id. I added another terraform setup with it so I was covering testing for both scenarios as they both exist.